content/200-concepts/300-more/050-environment-variables/200-using-multiple-env-files.mdx

```diff
@@ -7,38 +7,75 @@ tocDepth: 2
 
 <TopBlock>
 
-There is a risk that your production database could be deleted if you store different connection URLs to each of your environments within a single `.env` file.
+When developing an application you will generally have different environments such as `development`, `test` and `production` which you use to **separate the stages of the development cycle**.
 
-One solution is to have multiple `.env` files which each represent different environments. In practice this means you create a file for each of your environments:
+Prisma can access each environment via a [connection url](../../../reference/database-reference/connection-urls) which is stored in a `.env` file.
+
+</TopBlock>
+
+## Scenario: Running integration tests
+
+You have three [connection urls](../../../reference/database-reference/connection-urls) for each of your environments stored in a single `.env` file:
+
+```env file=.env
+
+// Connection URL to development environment
+DEVELOPMENT_DATABASE_URL="postgresql://prisma:prisma@localhost:5433/development"
+
+// Connection URL to testing environment
+TESTING_DATABASE_URL="postgresql://prisma:prisma@localhost:5433/test"
```

Comment:
Would be better to only use default port everywhere to not confuse users I think
```suggestion
DEVELOPMENT_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/development"

// Connection URL to testing environment
TESTING_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/test"
```